### PR TITLE
Add Free purchase transaction

### PR DESCRIPTION
**Pull request by pedromtec**
#### What problem is this solving?

Context [here](https://github.com/vtex-apps/order-payment/pull/13)

Free purchase transaction

#### How should this be manually tested?

Choose the account `pilateslovers`. 
[App Store](https://pedrocheckout--extensions.myvtex.com/vtex-facebook-pixel/p)

I'm getting some issues to test the flow using `checkoutio` account because I can't choose a free shipping.  
[Checkout IO](https://pedrocheckout--checkoutio.myvtex.com/Produto-gratis/p)

#### Checklist/Reminders

- [ ] Updated `README.md`.
- [x] Updated `CHANGELOG.md`.
- [ ] Linked this PR to a Jira story (if applicable).
- [ ] Updated/created tests (important for bug fixes).
- [ ] Deleted the workspace after merging this PR (if applicable).

#### Type of changes

✔️ | Type of Change
---|---
_ | Bug fix 
x | New feature 
_ | Breaking change 
_ | Technical improvements

### CODE REVIEW

**Review by arturpimentel — APPROVED** [review]

**Review by lucasecdb — COMMENTED** [review]
can you rebase?

**Review by klzns — CHANGES_REQUESTED** [review]
Rebase needed

File: node/package.json [hunk 1]
```diff
@@ -15,28 +15,28 @@
...
+    "vtex.css-handles": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.css-handles@0.4.4/public/@types/vtex.css-handles",
+    "vtex.device-detector": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.device-detector@0.2.5/public/@types/vtex.device-detector",
+    "vtex.flex-layout": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.flex-layout@0.15.1/public/@types/vtex.flex-layout",
     "vtex.order-coupon": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.order-coupon@0.7.1/public/@types/vtex.order-coupon",
-    "vtex.order-manager": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.order-manager@0.8.4/public/@types/vtex.order-manager",
-    "vtex.order-payment": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.order-payment@0.6.0/public/@types/vtex.order-payment",
+    "vtex.order-manager": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.order-manager@0.8.7/public/@types/vtex.order-manager",
+    "vtex.order-payment": "https://pedrocheckout--checkoutio.myvtex.com/_v/private/typings/linked/v1/vtex.order-payment@0.7.0+build1605552691/public/@types/vtex.order-payment",
```
Comment (klzns) [inline node/package.json]: Remove linked types

File: node/package.json [hunk 2]
```diff
@@ -15,28 +15,28 @@
...
-    "vtex.flex-layout": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.flex-layout@0.15.0/public/@types/vtex.flex-layout",
+    "vtex.checkout": "https://pedrocheckout--checkoutio.myvtex.com/_v/private/typings/linked/v1/vtex.checkout@2.7.1+build1605555284/public/@types/vtex.checkout",
+    "vtex.checkout-cart": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.checkout-cart@0.31.0/public/@types/vtex.checkout-cart",
+    "vtex.checkout-container": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.checkout-container@0.6.2/public/@types/vtex.checkout-container",
+    "vtex.checkout-graphql": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.checkout-graphql@0.47.0/public/@types/vtex.checkout-graphql",
+    "vtex.checkout-identification": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.checkout-identification@0.2.1/public/@types/vtex.checkout-identification",
+    "vtex.checkout-step-group": "https://pedrocheckout--checkoutio.myvtex.com/_v/private/typings/linked/v1/vtex.checkout-step-group@0.9.0+build1605554263/public/@types/vtex.checkout-step-group",
```
Comment (klzns) [inline node/package.json]: same

File: node/package.json [hunk 3]
```diff
@@ -15,28 +15,28 @@
...
-    "vtex.checkout-graphql": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.checkout-graphql@0.34.2/public/@types/vtex.checkout-graphql",
-    "vtex.checkout-identification": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.checkout-identification@0.2.0/public/@types/vtex.checkout-identification",
-    "vtex.checkout-step-group": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.checkout-step-group@0.8.0/public/@types/vtex.checkout-step-group",
-    "vtex.checkout-summary": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.checkout-summary@0.16.0/public/@types/vtex.checkout-summary",
-    "vtex.css-handles": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.css-handles@0.4.2/public/@types/vtex.css-handles",
-    "vtex.device-detector": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.device-detector@0.2.4/public/@types/vtex.device-detector",
-    "vtex.flex-layout": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.flex-layout@0.15.0/public/@types/vtex.flex-layout",
+    "vtex.checkout": "https://pedrocheckout--checkoutio.myvtex.com/_v/private/typings/linked/v1/vtex.checkout@2.7.1+build1605555284/public/@types/vtex.checkout",
```
Comment (klzns) [inline node/package.json]: same

**Review by klzns — CHANGES_REQUESTED** [review]

File: node/package.json (hunk 1 above)
Comment (pedromtec) [reply node/package.json]: OK, I intended to do it when I merge these PRs but there is not a dependency here.

Comment (pedromtec) [reply node/package.json]: There are 4 open PRs to this feature.

**Review by lucasecdb — APPROVED** [review]

**Review by klzns — APPROVED** [review]

**Pull request merged**